docs/antctl.md

```diff
@@ -57,16 +71,57 @@ one by setting the `KUBECONFIG` environment variable or with `--kubeconfig`
 The following sub-sections introduce a few commands which are useful for
 troubleshooting the Antrea system.
 
+### Collecting support information
+
+Starting with version 0.7.0, Antrea support the `antctl supportbundle` command,
+which can collect information from the cluster, the Antrea Controller and all
+Antrea agents. This information is useful when trying to troubleshoot issues in
+Kubernetes clusters using Antrea. In particular, when running the command
+out-of-cluster, all the information can be collected under one single directory,
+which you can upload and share when reporting issues on Github. Simply run the
+command as follows:
+
+```bash
+antctl supportbundle [-d <TARGET_DIR>]
+```
+
+If you omit to provide a directory, antctl will create one in the current
+working directory, using the current timestamp as a suffix. The command also
+provides additional flags to filter the results: run `antctl supportbundle
+--help` for the full list.
+
+The collected support bundle will include the following (more information may be
+included over time):
+ * cluster information: description of the different K8s resources in the
+   cluster (Nodes, Deployments, etc.).
+ * Antrea Controller information: all the available logs (contents will vary
+   based on the verbosity selected when running the controller) and state stored
+   at the controller (e.g.computed NetworkPolicy objects).
+ * Antrea Agent information: all the available logs from the agent and the OVS
+   daemons, network configuration of the Node (e.g. routes, iptables rules, OVS
+   flows) and state stored at the agent (e.g. computed NetworkPolicy objects
+   received from the controller).
+
+**Be aware that the generated support bundle includes a lot of information,
+  including logs, so please review the contents of the directory before sharing
+  it on Github and ensure that you do not share anything sensitive.**
+
+The `antctl supportbundle` command can also be run inside a Controller or Agent
```

Comment:
Do you want to mention we are able to collect bundles of only matched Nodes?

BTW, I suggested to review the current command design: provide a flag to collect controller bundle only; command with a matching expression should not include controller bundle.

Reply:
I agree with your points. I'm aware of the `--controller-only` flag. It seems that matching on Node name still collects cluster info & controller bundle.

I'm happy to update this document in the future when changes are made to the command. For now, I'd rather not include information that may become stale soon if the command is still evolving.

Reply:
Sounds good to me.